README.md

```diff
@@ -150,7 +147,7 @@ defmodule MyApp.SomeControllerPerhaps do
 end
 ```
 
-Your application is now set up to send email with Bamboo! :tada:
+Your application is now set up to send an email with Bamboo! :tada:
```

Comment:
I think this used to be correct before. The whole application was set up to send email (not just _an_ email) with Bamboo. Does that make sense?